package.json

```diff
@@ -51,7 +51,12 @@
     "test:e2e:ct:hl": "cd packages/commercetools/theme && yarn run test:e2e:hl",
```

Comment:
describe this in `contributing` part of docs please ;)

Reply:
I am not sure whether this need to be included in the contributing docs. I added note about it in our notion document on how to push and pull to/from local registry. This feature is mostly needed for the core team that want to make sure that the package we are about to release to global npm.registry works as expected. This is purely a Core Team functionality